sympy/plotting/utils.py

```diff
@@ -1,4 +1,11 @@
+from sympy.core.containers import Tuple
+from sympy.core.expr import Expr
 from sympy.core.function import AppliedUndef
+from sympy.core.relational import Relational
+from sympy.core.symbol import Dummy
+from sympy.core.sympify import sympify
+from sympy.logic.boolalg import BooleanFunction
+from sympy.physics.mechanics import Vector as MechVector
```

Comment:
The test failure is because you import mechanics here:
```
Error: The submodule sympy.combinatorics was imported with 'from sympy import *', but it was
not expected to be.

If sympy.combinatorics is a new module that has functions that are imported at the
top-level, then the whitelist in bin/test_submodule_imports should be updated.
If it is not, the place that imports it should be modified so that it does not
get imported at the top-level, e.g., by moving the 'import sympy.combinatorics' import
inside the function that uses it.

If you are unsure which code is importing sympy.combinatorics, it may help to add 'raise
Exception' to sympy/combinatorics/__init__.py and observe the traceback from
running 'from sympy import *'.
```
The physics module is not supposed to be imported by default with `from sympy import *` because it slows down the SymPy import time.

Reply:
Ok, I'll figure it out a way to avoid this import. Thank you.